Games/types/Mafia/Action.js

```diff
@@ -19,4 +19,20 @@ module.exports = class MafiaAction extends Action {
             }
         }
     }
+
+    getVisitors(player) {

```

Comment:
more flexible than enqueue visitors as you can enqueue visitors of other players

e.g. caroler wants to check if their target has visitors